torch/_inductor/codegen/wrapper.py

```diff
@@ -478,7 +478,18 @@ def codegen_input_size_asserts(self):
                 continue
             size = self.codegen_shape_tuple(buf.get_size())
             stride = self.codegen_shape_tuple(buf.get_stride())
-            self.prefix.writeline(f"assert_size_stride({name}, {size}, {stride})")
+            error_msg = """'''
+Found a size/stride mismatch. Try re-running with layout optimization disabled:
+You can do this with either:
+    `torch._inductor.config.layout_optimization = False` (programatically)
+    `TORCHINDUCTOR_LAYOUT_OPTIMIZATION=0` (env var)
+If you are using DDP, and the assert is still failing even with layout optimization disabled,
+then please additionally try disabling the DDP Optimizer:
+    `torch._dynamo.config.optimize_ddp = False`
```

Comment:
best to suggest (1) turn on the lazy ddp comple flag, then if that fails, (2) optimize_ddp=False

Reply:
Will do!